docs/ibis-for-pandas-users.ipynb

```diff
@@ -0,0 +1,4363 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    t = ibis.pandas.connect({'t': df}).table('t')
</pre><p>Is the goal to show how to do pandas things with ibis, or to show the pandas backend? Perhaps surprisingly, the pandas backend isn't the best way to show the full feature set of ibis and I'd recommend using <code>ibis.memtable(df)</code> to show ibis operations on <code>DataFrame</code>s.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/4715/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
The purpose was to show how to convert common pandas operations to Ibis operations. I figure pandas would have to be installed on their systems anyway, so they would be able to run the examples. I don't see `ibis.memtable` anywhere, is that a v4 thing?

Reply:
No, that should be available in 3.2.0. I'll create an issue to document it.

`ibis.memtable` does require `duckdb` to be installed to work transparently without setting `ibis.options.default_backend` so that's something to consider if you're trying to avoid additional installation steps (though duckdb is easy to install).